Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3-alpine
+
+ARG PHOEBE_BRANCH=master
+
+RUN apk add --no-cache \
+      git \
+      build-base \
+      musl-dev
+
+RUN git clone --branch ${PHOEBE_BRANCH} --single-branch --depth 1 \
+      https://github.com/phoebe-project/phoebe2.git /phoebe
+
+WORKDIR /phoebe
+
+RUN pip install --upgrade pip
+
+RUN pip install --no-cache-dir .
+
+RUN mkdir -p /workspace
+WORKDIR /workspace
```

Comment:
is this useful (or potentially harmful/unexpected) to anyone else or just the specific setup for the web servers?  If just us, maybe it should live in one of the server repos instead?

Reply:
this is just to avoid phoebe being imported from the source tree for reporting python/phoebe versions.

Reply:
The main reason for the base image is inheritance -- any image that requires phoebe will build from this image instead of having to create a fresh image, install all dependencies and build phoebe over and over again. This way people would simply do:
```
FROM phoebe2:blending
. . .
```
and be done. Eventually we may want to host this image on either dockerhub or github container hub.